api/app/users/tests.py

```diff
@@ -566,7 +566,7 @@ def test_response_and_not_invited(self):
 
         # Assert that request succeeds and that users that responded are returned.
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(len(response.json), 1)
+        self.assertEqual(len(response.json), 0)
```

Comment:
I think there should be users returned here. We  want to check that users that have responded (but are not invited guests) are returned. And the response should include that they are not invited guests (in the field we added).

Reply:
How do we specify the response that they are not invited guests?